### PR TITLE
Updated Django cheat sheets

### PR DESCRIPTION
**Pull request by Rituraj-13**
## What does this PR do?
Add a few Django cheat sheet resources.

### Description
This PR provides some useful Django Cheat Sheet links which can be opened with one click and doesn't ask for any verification or signup related issues.

### Why is this valuable (or not)?
The resources are beginner friendly and qualitative.

### How do we know it's really free?
These are either blog pages or PDFs which opens in one click and doesn't ask for any kind of verification and signup issues.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a cheat sheet.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Rituraj-13** [comment]
Here is the link of the [Issue](https://github.com/EbookFoundation/free-programming-books/issues/7106).

**Review by davorpa — CHANGES_REQUESTED** [review]
Should move the entire section to Python (as subsection), since is a Python framework. E.g.

```markdown
### Python

....

#### Django

....
```

**Comment by Rituraj-13** [comment]
@davorpa I have made the changes, kindly have a look.

**Review by davorpa — APPROVED** [review]
Looks well after my changes

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -259,6 +260,14 @@
 * [Scapy Cheat Sheet](https://www.templateroller.com/template/160817/scapy-cheat-sheet-jeremy-stretch.html) - Jeremy Stretch (PDF)
 
 
+#### Django
+
+* [All in one Cheatsheet of Django](https://cheatography.com/ogr/cheat-sheets/django/pdf/) - Cheatography (PDF)
+* [Beginner's Python Cheat Sheet – Django](https://edu.anarcho-copy.org/Programming%20Languages/Python/Python%20CheatSheet/beginners_python_cheat_sheet_pcc_django.pdf) - Eric Matthes (PDF)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:266]: This comes from https://ehmatthes.github.io/pcc_2e/cheat_sheets/cheat_sheets/ and that's what we should link. Please change.

Comment (davorpa) [reply more/free-programming-cheatsheets.md]: Done at 9b78d4564852b8ccf25bc3d1a16aaca2fc3e5910

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**